### ISSUE TITLE
Alter Mailing List Property 

### ISSUE THREAD

**Issue by jyd44** [ovh/php-ovh#64]
It seems that this call laucnh a new task. However, the result of the call is void. 
As such, there is no way to control the execution of this new task.
Is there a way to improve this API ?
(it seems that void return is the rule for any Put call, why ?)

Another point: the documentation available on https://api.ovh.com/console/#/email/domain/{domain}/mailingList/{name}#PUT is incorrect. 
$result = $ovh->put('/email/domain/{domain}/mailingList/{name}'); does not provide the new properties and is not in line with the prototype of the function.

**Comment by jyd44** [comment]
To emphasize my previous post, see below a "php code" and the resulting traces:
```$domain = "my_domain.net";
$ml = "ml_tests";
$ml_properties = $conn->get('/email/domain/'.$domain.'/mailingList/'.$ml );
echo time().": Get Properties =>"; print_r($ml_properties);
$properties = array();
$run_values = array('mailinglist', 'lastuser');
foreach ($run_values as $value)
{
	$properties['replyTo'] = $value;
	$status = 0; $no_call = 1;
	do
	{
		try
		{
			$status = 0;
			echo sprintf('%d: Call num %d' , time(), $no_call++);
			$on_going_task = $conn->get('/email/domain/'.$domain.'/task/mailinglist');
			if(count($on_going_task))
			{
				foreach ($on_going_task as $task_id)
					echo sprintf('Before: Tache en cours %d' , $task_id);
			}
			else echo "No on-going task";
			echo sprintf("Setting Property replyTo to: %s", $properties['replyTo']);
			$retour = $conn->put('/email/domain/'.$domain.'/mailingList/'.$ml, $properties );
			$on_going_task = $conn->get('/email/domain/'.$domain.'/task/mailinglist');
			foreach ($on_going_task as $task_id)
					echo sprintf('After: On-going Task %d' , $task_id);
		}
		catch(\GuzzleHttp\Exception\ClientException $e)
		{
			$response = $e->getResponse();
			$status = $response->getStatusCode();
			echo sprintf("Error on properties setting: %s, %s", $status, $response->getReasonPhrase()));
		}
		if ($status == 409) { echo 'Sleep 5 seconds'; sleep(5);}
	}while ($status == 409);
	$ml_properties = $conn->get('/email/domain/'.$domain.'/mailingList/'.$ml );
	echo time().": Get Properties =>"; print_r($ml_properties);
}
```

**Resulting TRACES:**
Starting from a state where replyTo is set to 'lastuser'
```
**Initial Call to Get Properties**
1502469252: Ml_Property Get =>
	Array ( [language] => fr 
		[options] => Array ( 
                                                [subscribeByModerator] => 1 
                                                [usersPostOnly] => 1 
                                                [moderatorMessage] => ) 
		[name] => ml_tests 
		[ownerEmail] => name@domain.net 
		[replyTo] => lastuser 
		[id] => 466251 
		[nbSubscribersUpdateDate] => 2017-08-11T18:32:06+02:00 
		[nbSubscribers] => 5 ) 
```
```
RUN 'mailinglist'
1502469269: Call number  1
No on-going task
Setting Property replyTo to: mailinglist
After: On-Going Task: 480081403

1502469269: Ml_Property Get =>
	Array ( 
		[language] => fr 
		[options] => Array ( [subscribeByModerator] => 1 [usersPostOnly] => 1 [moderatorMessage] => ) 
		[name] => ml_tests 
		[ownerEmail] => name@domain.net 
		[replyTo] => mailinglist 
		[id] => 466251 
		[nbSubscribersUpdateDate] => 2017-08-11T18:32:06+02:00 
		[nbSubscribers] => 5 ) 
```
The result of the call is immediately available, but a task as been launched.
```
RUN 'lastuser' 20 seconds later
1502469289:	Call number  1
		Before: On-Going Task: 480081403
		Setting Property replyTo to: lastuser
		Error on properties setting: 409, ObjectAlreadyExists
		Sleep  5 seconds
1502469295:	Call number  2
		Before: On-Going Task: 480081403
		Setting Property replyTo to: lastuser
		Error on properties setting: 409, ObjectAlreadyExists
		Sleep  5 seconds
1502469300:	Call number  3
		Before: On-Going Task: 480081403
		Setting Property replyTo to: lastuser
		Error on properties setting: 409, ObjectAlreadyExists
		Sleep  5 seconds
1502469305:	Call number  4
		Before: On-Going Task: 480081403
		Setting Property replyTo to: lastuser
		Error on properties setting: 409, ObjectAlreadyExists
		Sleep  5 seconds
1502469310:	Call number  5
		No on-going task
		Setting Property replyTo to: lastuser
		After: On-Going Task: 480081404

1502469311: Ml_Property Get =>
		Array ( 
			[language] => fr 
			[options] => Array ( [subscribeByModerator] => 1 [usersPostOnly] => 1 [moderatorMessage] => ) 
			[name] => ml_tests 
			[ownerEmail] => name@domain.net 
			[replyTo] => lastuser 
			[id] => 466251 
			[nbSubscribersUpdateDate] => 2017-08-11T18:35:07+02:00 
			[nbSubscribers] => 5 ) 
```

**Comment by rbeuque74** [comment]
Hello,
Closing as stalled. Feel free to reopen if still relevant.

Romain
